README.md

```diff
@@ -363,17 +361,17 @@ So that the generated HTML looks something like
 ```html
 <div class="gallery">
   <img
-    src="https://assets.imgix.net/examples/pione.jpg?auto=format&ar=16%3A9&"
+    src="https://assets.imgix.net/examples/pione.jpg?auto=format&ar=16%3"
     ...
   />
   <img
-    src="https://sdk-test.imgix.net/%D8%B3%D8%A7%D9%86%D8%AF%D9%88%DB%8C%DA%86.jpg?auto=format&ar=16%3A9&"
+    src="https://sdk-test.imgix.net/%D8%B3%D8%A7%D9%86%D8%AF%D9%88%DB%8C%DA%86.jpg?auto=format&ar=16%3"
```

Comment:
Looks like you trimmed off the `A9` as well, so this won't give us an encoded `:` anymore. Also, I just noticed that this example ([`8382f4a` (#799)](https://github.com/imgix/react-imgix/pull/799/commits/8382f4a5e4e32d53731deed5369a0effe5bac2e5#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R344)) needs `fit: crop` in order for the `ar` example to work.